### PR TITLE
add gov bravo config

### PR DESCRIPTION
**Pull request by coburncoburn**

### CODE REVIEW

**Review by torreyatcitty — APPROVED** [review]
lgtm

**Review by jflatow — APPROVED** [review]

File: networks/mainnet-abi.json [hunk 1]
```diff
@@ -1,4 +1,370 @@
 {
+    "GovernorBravo": [
```
Comment (torreyatcitty) [inline networks/mainnet-abi.json:2]: Actually looking at this, `GovernorBravo` doesn't have the propose function in this abi. How did we generate it?

**Review by torreyatcitty — CHANGES_REQUESTED** [review]
Left a comment on the GovernorBravo ABI

File: networks/mainnet-abi.json (hunk 1 above)
Comment (coburncoburn) [reply networks/mainnet-abi.json:2]: ah, was missing delegate functions

**Pull request merged**